pizza2.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Pizza-order</title>
+        <style>
+            nav{
+                background-color: yellow;
+                align-content: space-around;
+                padding-left: 60px;
+            }
+            a{
+                padding: 80px;  
+                
+            }
+            label{
+                color:skyblue;
+            }
+            [type="text"]{
+                border-color: green;
+            }
+            [type="email"]{
+                background-color: blue;
+            }
+            [type="submit"]{
+                background-color: green;
+            }
+            body{
+                background-color: salmon;
+            }
+            div{
+                border-style: solid;               
+            }
+            
+        </style>
+    </head>
+    <body>
+        <nav><h1>
+            <img src="https://unblast.com/wp-content/uploads/2018/09/Bubbles.jpg" height="60" width="60">
+            <a target="_blank" href="https://www.google.ro/" ><b><ins>Home</ins></b></a>
+            <a target="_blank" href="https://www.google.ro/"><b><ins>About Us</ins></b></a>
+            <a target="_blank" href="https://www.google.ro/"><b><ins>Our Clients</ins></b></a>
+            <a target="_blank" href="https://www.google.ro/"><b><ins>Contact Us</ins></b></a></h1>
```

Comment:
Nu este ok sa pui un h1 peste toate aceste nav-uri. Nu stiu ce ai vru sa pui asa. 
De asemenea, h1 se foloseste doar o data pentru a indica titlul principal al pagini, nu cateva link-uri
De asemenea, pentru ca este o lista de navigare, este mai ok sa o punem intr-un ul

```suggestion
        <nav>
            <ul>
                <li><img src="https://unblast.com/wp-content/uploads/2018/09/Bubbles.jpg" height="60" width="60"></li>
                <li> <a target="_blank" href="https://www.google.ro/" ><b><ins>Home</ins></b></a></li>
                <li> <a target="_blank" href="https://www.google.ro/"><b><ins>About Us</ins></b></a></li>
                <li> <a target="_blank" href="https://www.google.ro/"><b><ins>Our Clients</ins></b></a></li>
                <li> <a target="_blank" href="https://www.google.ro/"><b><ins>Contact Us</ins></b></a></h1></li>
            </ul>    
```